lib/generators/react_on_rails/templates/base/base/client/webpack.client.rails.config.js

```diff
@@ -6,6 +6,8 @@
 
 const webpack = require('webpack');
 const config = require('./webpack.client.base.config');
+const devBuild = process.env.NODE_ENV !== 'production';
+
```

Comment:
@justin808 ok I can add that in. It doesn't seem like `nodeEnv` is used in 
`ib/generators/react_on_rails/templates/base/base/client/webpack.client.rails.config.js`
although it is used in 
`lib/generators/react_on_rails/templates/base/base/client/webpack.client.base.config.js.tt`
(subject of my commit before this one)
Do you still want me to include that line?
Realizing now that perhaps I should have split these two things into two different PRs (?)